src/fastdiff.js

```diff
@@ -101,12 +101,13 @@ export default function fastDiff( a, b, cmp, atomicChanges = false ) {
 	};
 
 	// Transform text or any iterable into arrays for easier, consistent processing.
+	// Array.from was used here but it generated incorrect results for multi-byte unicode sequences.
```

Comment:
I'd rather explain why we use the cloning mechanism of of arrays over using Array.from. Something like:

> We convert the string to an array by using the `slice()` method because, unlike Array.from(), it returns single-byte array items. See ckeditor/ckeditor5#3147.
> 
> We need to make sure here that `fastDiff()` works identical to `diff()` here.